install/installer/pkg/components/spicedb/data/schema.yaml

```diff
@@ -5,43 +5,61 @@
 schema: |-
   definition user {}
 
+  // There's only one global installation
+  definition installation {
+
+    // only users that are not owned by an org are considered installation-level users
+    relation user: user
+    relation admin: user
+    permission is_admin = admin
+
+    // orgs can only be created by installation-level users
+    permission create_organization = user
+
+  }
+
   definition organization {
+    // Every organization is associated with the installation
+    relation installation: installation
+
     // Every user in an organization is automatically a member
     relation member: user
+    permission is_member = member + is_owner
     // Some users in an organization may additionally have the `owner` role
-    // granting them more privilidges with respect to the organization.
+    // granting them more privileges with respect to the organization.
     relation owner: user
+    permission is_owner = owner + installation->is_admin
 
     // General operations on organization
-    permission read_info = member + owner
-    permission write_info = owner
+    permission read_info = is_member
+    permission write_info = is_owner
 
     // Operations on Organization's Members
-    permission read_members = member + owner
-    permission invite_members = member + owner
-    permission write_members = owner
+    permission read_members = is_member
+    permission invite_members = is_member
+    permission write_members = is_owner
 
     // Any user in the organization can try to leave their organization
-    permission leave = owner + member
+    permission leave = is_member
 
     // Owners can delete the organization,
     // and through this process also disconnect all users from the organization
-    permission delete = owner
+    permission delete = is_owner
 
     // Only owners can create new projects
     permission create_project = owner
```

Comment:
I left `create_project` permission intentional only for `owner` as I don't see why an admin should create a project and also wanted to illustrate how we still have the possibility for fine-grained permission grants.